### ISSUE TITLE
README missing import datetime

### ISSUE THREAD

**Issue by jeppoo1** [pnuu/fmiopendata#5]
Hi,
if I understood correctly, your [(README )](https://github.com/pnuu/fmiopendata/blob/master/README.md) examples are missing an import.
I'm talking about this part:

Download and parse observation data
from fmiopendata.wfs import download_stored_query

Retrieve the latest hour of data from a bounding box
end_time = dt.datetime.utcnow()
start_time = end_time - dt.timedelta(hours=1)
Convert times to properly formatted strings
start_time = start_time.isoformat(timespec="seconds") + "Z"
-> 2020-07-07T12:00:00Z
end_time = end_time.isoformat(timespec="seconds") + "Z"
-> 2020-07-07T13:00:00Z

obs = download_stored_query("fmi::observations::weather::multipointcoverage",
                            args=["bbox=18,55,35,75",
                                  "starttime=" + start_time,
                                  "endtime=" + end_time])

It only seems to work, if I add `import datetime as dt` on top of it.
Am I correct about this?

@pnuu

**Comment by jeppoo1** [comment]
**Another comment on the README file:**
On the bottom part of the page, in this paragraph:

_The times are as a list of datetime objects
times = obs.data['Helsinki Malmi lentokenttä']['times']
Other data fields have another extra level, one for values and one for the unit
print(len(obs.data['Helsinki Malmi lentokenttä']['t2m']['values']))
-> 72
print(len(obs.data['Helsinki Malmi lentokenttä']['t2m']['values']))
-> 'degC'_

the last line (`print(len(obs.data['Helsinki Malmi lentokenttä']['t2m']['values']))`) does not return "degC" but "72", because it is the same line of code as above. So, which code should I use instead on the last line, for it to print "degC"?

**Comment by pnuu** [comment]
Thanks again for reporting!

Indeed the `import datetime as dt` is missing. For the second part, the correct key is `'units'`. I'll fix these later today.

**Comment by pnuu** [comment]
And the latter print should've been
```python
print(obs.data['Helsinki Malmi lentokenttä']['t2m']['unit'])
# -> 'degC'
```
to show that the unit of these observations.

Fixed by #7
